From efe495ba2c029e16aa664dc6c65b17e38ddde84e Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:24 -0400
Subject: [redhat] kabi: Adding symbol single_open (fs/seq_file.c)

Message-id: <089a1f8a952eb0747475b3b44cebf495bfa95634.1410510773.git.poros@redhat.com>
Patchwork-id: 96121
O-Subject: [PATCH RHEL7.1 204/233] kabi: Adding symbol single_open (fs/seq_file.c)
Bugzilla: 1062361
RH-Acked-by: Alexander Viro <aviro@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Needed by PowerPath to handle /proc entries created by it.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/single_open
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/single_open
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/single_open

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/single_open b/redhat/kabi/kabi-module/kabi_ppc64/single_open
new file mode 100644
index 0000000..b3ee98a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/single_open
@@ -0,0 +1,2 @@
+#1-
+0x30916d9f single_open vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/single_open b/redhat/kabi/kabi-module/kabi_s390x/single_open
new file mode 100644
index 0000000..782625c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/single_open
@@ -0,0 +1,2 @@
+#1-
+0xcf67f900 single_open vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/single_open b/redhat/kabi/kabi-module/kabi_x86_64/single_open
new file mode 100644
index 0000000..6426a82
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/single_open
@@ -0,0 +1,2 @@
+#1-
+0x16a5a12f single_open vmlinux EXPORT_SYMBOL
-- 
1.7.1